source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.h

```diff
@@ -63,7 +63,7 @@ class SwiftASTManipulatorBase {
 
     VariableInfo(CompilerType &type, swift::Identifier name,
                  VariableMetadataSP metadata, 
-                 swift::VarDecl::Specifier specifier = swift::VarDecl::Specifier::Let,
+                 swift::VarDecl::Specifier specifier, 
```

Comment:
Eh, sorry, for some reason I thought you had changed this as requested, and the window elided the "= ..." so I though you had removed the wrong part.

Anyway, please remove the variable before committing.